Diagnostic/diagnostic.py

```diff
@@ -25,36 +25,41 @@
 import traceback
 import xml.dom.minidom
 import xml.etree.ElementTree as ET
-from six import string_types
 
 import Utils.HandlerUtil as Util
 import Utils.LadDiagnosticUtil as LadUtil
 import Utils.XmlUtil as XmlUtil
 import Utils.ApplicationInsightsUtil as AIUtil
 from Utils.WAAgentUtil import waagent
 
-ExtensionShortName = 'LinuxAzureDiagnostic'
 WorkDir = os.getcwd()
 MDSDPidFile = os.path.join(WorkDir, 'mdsd.pid')
 MDSDPidPortFile = os.path.join(WorkDir, 'mdsd.pidport')
 OutputSize = 1024
 EnableSyslog = True
 waagent.LoggerInit('/var/log/waagent.log','/dev/stdout')
-waagent.Log("%s started to handle." %(ExtensionShortName))
+waagent.Log("LinuxAzureDiagnostic started to handle.")
 hutil = Util.HandlerUtility(waagent.Log, waagent.Error)
 hutil.try_parse_context()
+hutil.set_verbose_log(False)  # Explicitly set verbose flag to False. This is default anyway, but it will be made explicit and logged.
 public_settings = hutil.get_public_settings()
 private_settings = hutil.get_protected_settings()
 if not public_settings:
     public_settings = {}
 if not private_settings:
     private_settings = {}
-
+ExtensionOperationType = None
+# Better deal with the silly waagent typo, in anticipation of a proper fix of the typo later on waagent
```

Comment:
have you brought this up in the agent repo? If there's a fix we need to take, we can take it to the upcoming v2.1.5 release which we'll push to publishers soon.